Source/WebCore/editing/Editor.cpp

```diff
@@ -4056,14 +4056,14 @@ static RefPtr<Node> findFirstMarkable(Node* startingNode)
 
 bool Editor::selectionStartHasMarkerFor(DocumentMarker::MarkerType markerType, int from, int length) const
 {
-    auto node = findFirstMarkable(m_document.selection().selection().start().deprecatedNode());
+    auto node = findFirstMarkable(RefPtr { m_document.selection().selection().start().deprecatedNode() }.get());
```

Comment:
I continue to find this syntax so unpleasant I want us to refactor to avoid it. I think we can maybe change more functions to return or take `RefPtr` which makes a little extra churn when we call multiple functions on the same value but avoids this unpleasant idiom.